backend/plugin/advisor/advisor.go

```diff
@@ -114,6 +114,9 @@ const (
 	// MySQLColumnDisallowChanging is an advisor type for MySQL disallow CHANGE COLUMN statement.
 	MySQLColumnDisallowChanging Type = "bb.plugin.advisor.mysql.column.disallow-change"
 
+	// MySQLColumnDisallowDropIndex is an advisor type for MySQL disallow CHANGE COLUMN statement.
+	MySQLColumnDisallowDropIndex Type = "bb.plugin.advisor.mysql.column.disallow-drop-index"
```

Comment:
How about "bb.plugin.advisor.mysql.column.disallow-drop-in-index"?

Reply:
sounds good👌

Reply:
OK